server/src/main/java/org/opensearch/action/search/ListPitInfo.java

```diff
@@ -60,4 +54,30 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeLong(creationTime);
         out.writeLong(keepAlive);
     }
+
+    static final ConstructingObjectParser<ListPitInfo, Void> PARSER = new ConstructingObjectParser<>(
+        "list_pit_info",
+        true,
+        args -> new ListPitInfo((String) args[0], (long) args[1], (long) args[2])
+    );
+
+    private static final ParseField CREATION_TIME = new ParseField("creation_time");
+    private static final ParseField PIT_ID = new ParseField("pit_id");
+    private static final ParseField KEEP_ALIVE = new ParseField("keep_alive");
+    static {
+        PARSER.declareString(constructorArg(), new ParseField(PIT_ID.getPreferredName()));
```

Comment:
Why do you need to recreate `ParseField` again since you already have them?

``
PARSER.declareString(constructorArg(), PIT_ID);
PARSER.declareString(constructorArg(), CREATION_TIME);
PARSER.declareString(constructorArg(), KEEP_ALIVE);
```